corehq/apps/hqmedia/views.py

```diff
@@ -289,6 +293,69 @@ def update_multimedia_paths(request, domain, app_id):
     })
 
 
+@method_decorator(toggles.BULK_UPDATE_MULTIMEDIA_PATHS.required_decorator(), name='dispatch')
+@method_decorator(require_can_edit_apps, name='dispatch')
+class MultimediaTranslationsCoverageView(BaseMultimediaTemplateView):
+    urlname = "multimedia_translations_coverage"
+    template_name = "hqmedia/translations_coverage.html"
+    page_title = ugettext_noop("Translations Coverage")
+
+    @use_select2_v4
+    def dispatch(self, request, *args, **kwargs):
+        return super(MultimediaTranslationsCoverageView, self).dispatch(request, *args, **kwargs)
+
+    @property
+    def parent_pages(self):
+        return [{
+            'title': _("Multimedia Reference Checker"),
+            'url': reverse(MultimediaReferencesView.urlname, args=[self.domain, self.app.get_id]),
+        }]
+
+    @property
+    def page_context(self):
+        context = super(MultimediaTranslationsCoverageView, self).page_context
+        context.update({
+            "media_types": {t: CommCareMultimedia.get_doc_class(t).get_nice_name()
+                            for t in CommCareMultimedia.get_doc_types()},
+            "selected_langs": self.request.POST.getlist('langs', []),
+            "selected_media_types": self.request.POST.getlist('media_types', ['CommCareAudio', 'CommCareVideo']),
+        })
+        return context
+
+    def post(self, request, *args, **kwargs):
+        error = False
+
+        langs = request.POST.getlist('langs')
+        if not langs:
+            error = True
+            messages.error(request, "Please select a language.")
+
+        media_types = request.POST.getlist('media_types')
```

Comment:
Should not one be allowed to check for all media types if none are provided?

Reply:
Hmm...I see where you're coming from, but it isn't a pattern we use elsewhere, and there are at most 3 types to pick, so I'm going to leave this as-is.